ERCS/erc-7661.md

```diff
@@ -0,0 +1,103 @@
+---
+eip: 7661
```

Comment:
```suggestion
eip: 7662
```
Assigning next sequential EIP/ERC/RIP number.  Numbers are assigned by editors & associates, please don't self assign.
Numbering changed to sequential from 7500.

Please also update the filename.

Reply:
Apologies - the green checkmark seeker in me took over. I've updated the filename correctly.